index.js

```diff
@@ -133,8 +146,8 @@ module.exports = {
           run: utils.run,
           functionsDir: functionsBuildDir,
           secretsPath: join(process.cwd(), 'env.json'),
-          timeout: config.nimbella.timeout || 6000, // Default is 10 seconds
-          memory: config.nimbella.memory || 256 // Default is 256MB (max for free tier)
+          timeout: options.timeout, // Default is 10 seconds
```

Comment:
Comment says 10 seconds, but value in the default options is 6s. 
```suggestion
          timeout: options.timeout, // Default is 6 seconds
```